shared/common-adapters/search-filter.tsx

```diff
@@ -140,15 +141,17 @@ class SearchFilter extends React.PureComponent<Props, State> {
       this.focusOnMount()
     }
   }
+
+  private hotKeyToHotKeys = memoize((hotkey: string) => {
+    return [`mod+${hotkey}`]
+  })
+
   private keyHandler() {
     return (
       !Styles.isMobile &&
       this.props.hotkey &&
       !this.props.onClick && (
-        <Kb.HotKey
-          onHotKey={this.onHotkey}
-          hotKeys={[(Platforms.isDarwin ? 'command+' : 'ctrl+') + this.props.hotkey]}
-        />
+        <Kb.HotKey onHotKey={this.onHotkey} hotKeys={this.hotKeyToHotKeys(this.props.hotkey)} />
```

Comment:
why not just ```hotKeys={`mod+${this.props.hotkey}`}```?

Reply:
cause i touched this before i knew about the mod thing, i'll update